pkg/storage/testing/chunk.go

```diff
@@ -39,15 +39,13 @@ func GenerateTestRandomChunk() swarm.Chunk {
 }
 
 // GenerateTestRandomSoChunk generates a valid single owner chunk.
-func GenerateTestRandomSoChunk(tb testing.TB) swarm.Chunk {
+func GenerateTestRandomSoChunk(tb testing.TB, cac swarm.Chunk) swarm.Chunk {
```

Comment:
I'd suggest improving the naming here, especially `...SoChunk`. Also `cac`, or it should be described in the function documentation.